tests/functional-tests/src/test/groovy/pages/app/modal/ConfirmRegisterAsWellPumpInstallerModal.groovy

```diff
@@ -0,0 +1,22 @@
+package pages
+
+class ConfirmRegisterAsPumpInstallerModal extends BaseAppPage {
+  static at = { modalModule.isOpen(modalSelector) }
+  static content = {
+    modalSelector(wait:true) { modalModule.modalWindow.has('h5', text:'Confirm register as Well Pump Installer') } //TODO add id if possible
```

Comment:
Not sure what this Well thing is - copypasta from different project I presume?

Reply:
Yes, place holder for now. I've included it for when we have to deal with modals.